### ISSUE TITLE
[NBug] An error occurred trying to start process 'https://gerrit.e...

### ISSUE THREAD

**Issue by BBzzz** [gitextensions/GitExtensions.GerritPlugin#62]
### Environment

- Git Extensions 4.0.0.15569
- Build cba315df5366fca01240d93b6e5a5077fd5e6630
- Git 2.38.1.windows.1
- Microsoft Windows NT 10.0.19042.0
- .NET 6.0.11
- DPI 96dpi (no scaling)

### Issue description

```
System.ComponentModel.Win32Exception (2): An error occurred trying to start process 'https://gerrit.ext.net.nokia.com/gerrit/c/CSF-CMT/+/4969386' with working directory 'C:\Users\bborbely\work\CSF-CMT'. The system cannot find the file specified.
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start(String fileName)
   at GitExtensions.GerritPlugin.FormGerritChangeSubmitted.<>c__DisplayClass1_0.<ShowSubmitted>b__0(Object s, EventArgs e)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.Label.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)
```

Just trying to create a new gerrit review

### Steps to reproduce

Try to publish a new gerrit review

### Did this work in previous version of GitExtensions?

Sure, it worked!

**Comment by lekhmanrus** [comment]
Just checked with:
- Git Extensions 4.0.0.15569
- Build cba315df5366fca01240d93b6e5a5077fd5e6630
- Git 2.39.0.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.12
- DPI 96dpi (no scaling)
- GitExtensions.GerritPlugin 2.0.1.128

And it works fine for me.
I'm closing this issue for now - if you still can reproduce that issue - please respond.

Thanks!

**Comment by tank09** [comment]
Hi,

we have the same issue after upgrading to git extension 4.0.2.16100 with plugin v2.0.4.177 
we tried on 3 different machines.

Windows 10, 22H2
.NET 6.0.13.32001

Which information do you need to check ?

Regards

**Comment by tank09** [comment]
Hi,

Any news on that ?
Do you need additional info ?

Thanks

**Comment by pmiossec** [comment]
@tank09 No.

@lekhmanrus It seems to be the same reason than for https://github.com/gitextensions/gitextensions/issues/10726#issuecomment-1430184815

The 3 occurrences of `Process.Start()` ( https://github.com/gitextensions/GitExtensions.GerritPlugin/search?q=Process.Start ) must be adapted to be compatible with the way to open a link with .net core.

It should be fixed like this: https://github.com/dotnet/core/issues/4109#issuecomment-573307566

Maybe you will like to call `OsShellUtil.OpenUrlInDefaultBrowser(()` like I did to fix the same problem in https://github.com/gitextensions/gitextensions/pull/10727

**Comment by lekhmanrus** [comment]
Closed in #69. Thanks @poupounetjoyeux! 🙂

**Comment by pmiossec** [comment]
> Closed in #69. Thanks @poupounetjoyeux! 🙂

@poupounetjoyeux va être très très joyeux ! 😁 (Bon, quand la release sera faite....)

**Comment by poupounetjoyeux** [comment]
Thanks @lekhmanrus I am now "Super Joyeux"!
